docker-compose-linux.yml

```diff
@@ -0,0 +1,67 @@
+# Allows us to start the stack without touching too much of our local system
```

Comment:
docker-compose can stay in forms-flow-ai/forms-flow-forms folder

Reply:
ok Wil Update , We can have a single docker-compose in this formio Repo so we can build the forms in here as well . We can have a CI/ CD process to get the image pushed to dockerhub and that image will be used by the forms-flow-ai Git Repo